ng-dev/release/publish/external-commands.ts

```diff
@@ -36,8 +37,15 @@ import {FatalReleaseActionError} from './actions-error';
 /**
  * Invokes the `ng-dev release set-dist-tag` command in order to set the specified
  * NPM dist tag for all packages in the checked out branch to the given version.
+ *
+ * Optionally, the NPM dist tag update can be skipped for experimental packages. This
+ * is useful when tagging long-term-support packages within NPM.
  */
-export async function invokeSetNpmDistCommand(npmDistTag: NpmDistTag, version: semver.SemVer) {
+export async function invokeSetNpmDistCommand(
```

Comment:
As I am looking at this, it seems odd that this is the place where we are making the decision about if an experimental package should be updated.

Does it make more sense to do this check before we invoke this command instead of invoking a command just to potentially tell it to skip itself.

Reply:
We need to do it there because the  configuration of experimental packages is local to the currently checked-out branch.

i.e. calling this command on an LTS branch should cause packages to be updated that e.g. no longer exist in master, and vice-verca

Reply:
It's also worth noting that this command updates all packages in a branch by default, so filtering beforehand would not work because we either would need to skip the whole npm dist update, or update all (even experimental ones). I think this is the right way to proceed here.